src/js/controllers/mapController.ts

```diff
@@ -12,11 +14,13 @@ interface ZoomParams {
 
 export class MapController {
   _map: Map | null;
-  _mapview: MapView | null;
+  _mapview: MapView | null | any;
```

Comment:
why `any`? that will prevent us from utilizing esri baked in types for both `mapview` and `sketchVM`